include/eredis.hrl

```diff
@@ -19,7 +19,7 @@
 -type options() :: [option()].
 -type server_args() :: options().               % for backwards compatibility
 
--type return_value() :: undefined | binary() | [binary() | nonempty_list()].
+-type return_value() :: undefined | binary() | [binary() | nonempty_list() | undefined].
```

Comment:
This type is far from perfect, but it's good to at least silence the warnings. OK.